### PR TITLE
RichCommand: PermanentlyDelete

### PR DESCRIPTION
**Pull request by ferrariofilippo**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #11481

**Notes**
This PR also removes some `KeyboardAccelerators` leftovers (`Refresh` and `Delete`)

**Validation**
How did you test these changes?
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [x] Did you implement any design changes to an existing feature?
   - [x] Was this change approved?
- [x] Are there any other steps that were used to validate these changes?
   1. Open app and select a file
   2. Use `Shift + Del` to permanently delete it or `Ctrl + D` to delete it

### CODE REVIEW

**Comment by yaira2** [comment]
@hishitetsu fyi

File: src/Files.App/Actions/FileSystem/PermanentlyDeleteAction.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+using Files.App.Extensions;
+using System.Threading.Tasks;
+
+namespace Files.App.Actions
+{
+	internal class PermanentlyDeleteAction : BaseDeleteAction, IAction
+	{
+		public string Label { get; } = "PermanentlyDelete".GetLocalizedResource();
+
+		public string Description { get; } = "PermanentlyDeleteDescription".GetLocalizedResource();
```
Comment (hishitetsu) [inline src/Files.App/Actions/FileSystem/PermanentlyDeleteAction.cs:11]:
```suggestion
	internal class DeleteItemPermanentlyAction : BaseDeleteAction, IAction
	{
		public string Label { get; } = "DeletePermanently".GetLocalizedResource();

		public string Description { get; } = "DeleteItemPermanentlyDescription".GetLocalizedResource();
```

File: src/Files.App/Commands/Manager/CommandManager.cs [hunk 2]
```diff
@@ -59,6 +59,7 @@ internal class CommandManager : ICommandManager
 		public IRichCommand PasteItem => commands[CommandCodes.PasteItem];
 		public IRichCommand PasteItemToSelection => commands[CommandCodes.PasteItemToSelection];
 		public IRichCommand DeleteItem => commands[CommandCodes.DeleteItem];
+		public IRichCommand PermanentlyDelete => commands[CommandCodes.PermanentlyDelete];
```
Comment (hishitetsu) [inline src/Files.App/Commands/Manager/CommandManager.cs:62]:
```suggestion
		public IRichCommand DeleteItemPermanently => commands[CommandCodes.DeleteItemPermanently];
```

File: src/Files.App/Commands/CommandCodes.cs [hunk 3]
```diff
@@ -25,6 +25,7 @@ public enum CommandCodes
 		PasteItem,
 		PasteItemToSelection,
 		DeleteItem,
+		PermanentlyDelete,
```
Comment (hishitetsu) [inline src/Files.App/Commands/CommandCodes.cs:28]:
```suggestion
		DeleteItemPermanently,
```

File: src/Files.App/Commands/Manager/CommandManager.cs [hunk 4]
```diff
@@ -199,6 +200,7 @@ public CommandManager()
 			[CommandCodes.PasteItem] = new PasteItemAction(),
 			[CommandCodes.PasteItemToSelection] = new PasteItemToSelectionAction(),
 			[CommandCodes.DeleteItem] = new DeleteItemAction(),
+			[CommandCodes.PermanentlyDelete] = new PermanentlyDeleteAction(),
```
Comment (hishitetsu) [inline src/Files.App/Commands/Manager/CommandManager.cs:203]:
```suggestion
			[CommandCodes.DeleteItemPermanently] = new DeleteItemPermanentlyAction(),
```

File: src/Files.App/Commands/Manager/ICommandManager.cs [hunk 5]
```diff
@@ -27,6 +27,7 @@ public interface ICommandManager : IEnumerable<IRichCommand>
 		IRichCommand PasteItem { get; }
 		IRichCommand PasteItemToSelection { get; }
 		IRichCommand DeleteItem { get; }
+		IRichCommand PermanentlyDelete { get; }
```
Comment (hishitetsu) [inline src/Files.App/Commands/Manager/ICommandManager.cs:30]:
```suggestion
		IRichCommand DeleteItemPermanently { get; }
```

File: src/Files.App/Strings/en-US/Resources.resw [hunk 6]
```diff
@@ -3184,4 +3184,10 @@
   <data name="ShowCheckboxesWhenSelectingItems" xml:space="preserve">
     <value>Show checkboxes when selecting items</value>
   </data>
+  <data name="PermanentlyDelete" xml:space="preserve">
+    <value>Permanently delete</value>
+  </data>
+  <data name="PermanentlyDeleteDescription" xml:space="preserve">
+    <value>Permanently delete item(s)</value>
+  </data>
```
Comment (hishitetsu) [inline src/Files.App/Strings/en-US/Resources.resw:3192]:
```suggestion
  <data name="DeletePermanently" xml:space="preserve">
    <value>Delete permanently</value>
  </data>
  <data name="DeleteItemPermanentlyDescription" xml:space="preserve">
    <value>Delete item(s) permanently</value>
  </data>
```

**Review by hishitetsu — COMMENTED** [review]
Action names should begin with a verb.
Can you update [CommandList.md](https://github.com/files-community/Files/blob/a117ff3b0b7f1eacdf3035b6fd5895562805dbce/specs/RichCommand/CommandList.md) as well?

**Comment by hishitetsu** [comment]
Can we make it possible to perform permanent deletion when pressing the delete button on the toolbar or flyout while pressing `shift`? File Explorer does.

**Comment by yaira2** [comment]
@hishitetsu can you open a feature request to discuss this idea?

**Comment by hishitetsu** [comment]
> @hishitetsu can you open a feature request to discuss this idea?

Opened.
https://github.com/files-community/Files/issues/12117

**Comment by ferrariofilippo** [comment]
> Action names should begin with a verb. Can you update [CommandList.md](https://github.com/files-community/Files/blob/a117ff3b0b7f1eacdf3035b6fd5895562805dbce/specs/RichCommand/CommandList.md) as well?

Done

**Review by hishitetsu — APPROVED** [review]
Thank you. LGTM

File: specs/RichCommand/CommandList.md [hunk 7]
```diff
@@ -20,6 +22,7 @@ This is the list of all commands defined in `CommandCodes` enum except `None`.
 |                    | PasteItem                           | Paste                                     | Paste item(s) from clipboard to current folder           | Ctrl+V              |
 |                    | PasteItemToSelection                | Paste                                     | Paste item(s) from clipboard to selected folder          | Ctrl+Shift+V        |
 |                    | DeleteItem                          | Delete                                    | Delete item(s)                                           | Delete              |
```
Comment (hishitetsu) [inline specs/RichCommand/CommandList.md:24]:
```suggestion
|                    | DeleteItem                          | Delete                                    | Delete item(s)                                           | Delete, Ctrl+D      |

```

**Review by hishitetsu — APPROVED** [review]

**Pull request merged**